darli/quaternions/quaternions.py

```diff
@@ -53,3 +53,37 @@ def L(quat, math: ArrayLikeFactory = NumpyLikeFactory):
     L[1:4, 1:4] = s * math.eye(3).array + hat(v, math)
 
     return L
+
+
+def G(quat, math: ArrayLikeFactory = NumpyLikeFactory):
+    """
+    Generate the matrix for quaternion and angular velocity multiplication.
+
+    Parameters:
+    q (npt.ArrayLike): The quaternion.
+
+    Returns:
+    np.ndarray: The matrix for quaternion and angular velocity multiplication.
+    """
+
+    return L(quat, math) @ H(math)
+
+
+def E(state, math: ArrayLikeFactory = NumpyLikeFactory):
+    """
+    Construct the state transition matrix for the quadrotor system.
```

Comment:
Remove "quadrotor" from docstring, this should be done for generilized state, also it is better to give a full names for the functions in quaternion: i.e: state_transition